src/m365/entra/commands/multitenant/multitenant-remove.ts

```diff
@@ -0,0 +1,155 @@
+import GlobalOptions from '../../../../GlobalOptions.js';
+import { Organization } from '@microsoft/microsoft-graph-types';
+import { Logger } from '../../../../cli/Logger.js';
+import request, { CliRequestOptions } from '../../../../request.js';
+import GraphCommand from '../../../base/GraphCommand.js';
+import commands from '../../commands.js';
+import { odata } from '../../../../utils/odata.js';
+import { cli } from '../../../../cli/cli.js';
+
+interface MultitenantOrganizationMember {
+  tenantId?: string;
+}
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  force?: boolean;
+}
+
+class EntraMultitenantRemoveCommand extends GraphCommand {
+  public get name(): string {
+    return commands.MULTITENANT_REMOVE;
+  }
+
+  public get description(): string {
+    return 'Removes a multitenant organization';
+  }
+
+  constructor() {
+    super();
+
+    this.#initTelemetry();
+    this.#initOptions();
+  }
+
+  #initTelemetry(): void {
+    this.telemetry.push((args: CommandArgs) => {
+      Object.assign(this.telemetryProperties, {
+        force: !!args.options.force
+      });
+    });
+  }
+
+  #initOptions(): void {
+    this.options.unshift(
+      {
+        option: '-f, --force'
+      }
+    );
+  }
+
+  public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
+
+    const removeMultitenantOrg = async (): Promise<void> => {
+      try {
+        const tenantId = await this.getCurrentTenantId();
+
+        let tenantsId = await this.getAllTenantsIds();
+        const tenantsCount = tenantsId.length;
+
+        const tasks: Promise<void>[] = [];
+
+        if (tenantsCount > 0) {
+          tenantsId.filter(x => x !== tenantId).forEach(t => {
+            tasks.push(this.removeTenant(logger, t));
+          });
+
+          await Promise.all(tasks);
```

Comment:
This can be simplified a bit

```suggestion
        if (tenantsCount > 0) {
                const tasks = tenantsId
                    .filter(x => x !== tenantId)
                    .map(t => this.removeTenant(logger, t));

                await Promise.all(tasks);
```